app/src/main/java/com/framgia/toeic/screen/main/MainActivity.java

```diff
@@ -0,0 +1,133 @@
+        NavigationView.OnNavigationItemSelectedListener {
+    private Toolbar mToolbar;
+    private NavigationView mNavigationView;
+    private ViewFlipper mViewFlipper;
+    private ProgressBar mProgressVocabulary;
+    private ProgressBar mProgressGrammar;
+    private ProgressBar mProgressBasicTest;
+    private ProgressBar mProgressExam;
+    private DrawerLayout mDrawer;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    protected int getLayoutResource() {
+        return R.layout.activity_main;
+    }
+
+    @Override
+    protected void initComponent() {
+        mToolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(mToolbar);
+        mDrawer = (DrawerLayout) findViewById(R.id.drawer_layout);
+        ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this, mDrawer, mToolbar,
+                R.string.navigation_drawer_open, R.string.navigation_drawer_close);
+        mDrawer.addDrawerListener(toggle);
+        toggle.syncState();
+        mNavigationView = findViewById(R.id.nav_view);
+        mNavigationView.setNavigationItemSelectedListener(this);
+        mViewFlipper = findViewById(R.id.viewflipper);
+        mProgressVocabulary = findViewById(R.id.progress_vocabulary);
+        mProgressGrammar = findViewById(R.id.progress_grammar);
+        mProgressBasicTest = findViewById(R.id.progress_basic_test);
+        mProgressExam = findViewById(R.id.progress_exam);
+    }
+
+    @Override
+    protected void initData() {
+        slideAdvertise();
+        mProgressVocabulary.getProgressDrawable()
+                .setColorFilter(getResources().getColor(R.color.material_orange_300),
+                        android.graphics.PorterDuff.Mode.MULTIPLY);
+        mProgressGrammar.getProgressDrawable()
+                .setColorFilter(getResources().getColor(R.color.material_brown_300),
+                        android.graphics.PorterDuff.Mode.MULTIPLY);
+        mProgressBasicTest.getProgressDrawable()
+                .setColorFilter(getResources().getColor(R.color.material_blue_accent_400),
+                        android.graphics.PorterDuff.Mode.MULTIPLY);
+        mProgressExam.getProgressDrawable()
+                .setColorFilter(getResources().getColor(R.color.material_red_accent_200),
+                        android.graphics.PorterDuff.Mode.MULTIPLY);
+    }
+
+    @Override
+    public void onBackPressed() {
+        mDrawer = (DrawerLayout) findViewById(R.id.drawer_layout);
+        if (mDrawer.isDrawerOpen(GravityCompat.START)) {
+            mDrawer.closeDrawer(GravityCompat.START);
+        } else {
+            super.onBackPressed();
+        }
+    }
+
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
+        int id = item.getItemId();
+        switch (id) {
+            case R.id.nav_vocabulary: {
```

Comment:
switch case k cần block code 
```suggestion
            case R.id.nav_vocabulary: 
```